src/components/ui/ResizablePane.js

```diff
@@ -0,0 +1,18 @@
+import { Resizable } from 're-resizable'
+import styled from 'styled-components'
+
+const ResizablePane = styled(Resizable).attrs((props) => ({
+  // Mark defaultWidth as transient prop with $ so that it's not passed to the div
+  defaultSize: { width: props.$defaultWidth, height: '100%' },
+  enable: { right: true }, // enable resizing only to the right
```

Comment:
Should `enable` also be passed as a prop? In case we happen to reuse ResizablePane somewhere.

Reply:
Ah yeah this is a good point! So I would remove line 7 and styled will automatically pass it through.